### PR TITLE
Fix Jacoco reporting in Nessie Quarkus projects

### PR DESCRIPTION
**Pull request by snazy**
Jacoco report generation via Quarkus is not so great, there were a bunch of issues, some breaking the build (flaky)

This change disables Jacoco report generation by Quarkus and changes it to let the Jacoco report task generate the reports.

### CODE REVIEW

File: servers/quarkus-cli/build.gradle.kts [hunk 1]
```diff
@@ -107,6 +110,41 @@ tasks.withType<QuarkusBuild>().configureEach {
...
+      // Must delete the Jacoco data file before running tests, because
+      // quarkus.jacoco.reuse-data-file=true in application.properties.
+      file("${project.buildDir}/jacoco-quarkus.exec").delete()
+      var reportDir = file("${project.buildDir}/jacoco-report")
+      delete { delete(reportDir) }
+      reportDir.mkdirs()
+    }
+  }
+
+val jacocoReport by
```
Comment (dimas-b) [inline servers/quarkus-cli/build.gradle.kts:125]: just wondering if it is too much trouble to aggregate all jacoco reports in our build and produce one overall _local_ report (may be in a separate PR) :thinking:

**Review by dimas-b — APPROVED** [review]

File: servers/quarkus-cli/build.gradle.kts (hunk 1 above)
Comment (snazy) [reply servers/quarkus-cli/build.gradle.kts:125]: We have that already via `codeCoverageReport`, right?

Comment (dimas-b) [reply servers/quarkus-cli/build.gradle.kts:125]: TIL :)
